### DISCUSSION TITLE
Tab to User Highlights separated from /mentions

### DISCUSSION THREAD

**Discussion by Pafon22** [Chatterino/chatterino2#4505] [Ideas]
Option to separate user highlights from /mentions tab. I would like /mentions to be just for my mentions (stuff I put in highlights like my mentions and regex). Users' highlights can be a separate tab, so if a user spams or uses their account as a channel bot, it won't affect my /mentions.
An interesting option would be to open filter options inside /mentions in "More":
-highlights enable/disable
-user highlights enable/disable
-specify highlights: -select the lines from highlights list
-specify user highlights: -select the lines from user highlight list
-ignore specify highlights: -select the lines from highlights list
-ignore specify user highlights: -select the lines from user highlight list

**Comment by Felanbird** [comment]
This can be obtained using Filters
https://wiki.chatterino.com/Filters/

If you're unable to get one that works I'll post one here when I have time
